tests/encryption_/tests.py

```diff
@@ -27,7 +27,7 @@ def test_encrypted_fields_map(self):
                 {
                     "path": "ssn",
                     "bsonType": "string",
-                    "queries": [{"contention": 1, "queryType": "equality"}],
+                    "queries": {"contention": 1, "queryType": "equality"},
```

Comment:
FYI it's a [list in the tutorial](https://github.com/mongodb/docs/blob/adad2b1ae41ec81a6e5682842850030813adc1e5/source/includes/qe-tutorials/python/queryable_encryption_tutorial.py#L45). Also for fields that support multiple query types I think it has to be a list of dictionaries, but maybe for a single query type this is OK.

Reply:
Nevermind! The `QueryType` class returns a `dict` so the user can decide to put one or more query types into a `list` or not use the `QueryType` class at all.